### PR TITLE
fix: churner message

### PR DESCRIPTION
**Pull request by 0x0aa0**

### CODE REVIEW

**Comment by gpsanant** [comment]
test failing but lgtm

File: src/RegistryCoordinator.sol [hunk 1]
```diff
@@ -889,19 +891,20 @@ contract RegistryCoordinator is
...
     function calculateOperatorChurnApprovalDigestHash(
+        address registeringOperator,
         bytes32 registeringOperatorId,
         OperatorKickParam[] memory operatorKickParams,
         bytes32 salt,
         uint256 expiry
     ) public view returns (bytes32) {
         // calculate the digest hash
-        return _hashTypedDataV4(keccak256(abi.encode(OPERATOR_CHURN_APPROVAL_TYPEHASH, registeringOperatorId, operatorKickParams, salt, expiry)));
+        return _hashTypedDataV4(keccak256(abi.encode(OPERATOR_CHURN_APPROVAL_TYPEHASH, registeringOperator, registeringOperatorId, operatorKickParams, salt, expiry)));
```
Comment (jianoaix) [inline src/RegistryCoordinator.sol:907]: Does this need to be identical with the offchain code? There are difference between the two

Comment (0x0aa0) [reply src/RegistryCoordinator.sol:907]: Ya they should be the same could you point me to where this hash in constructed offchain

**Review by mooselumph — APPROVED** [review]
lgtm

**Pull request merged**